plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -299,8 +299,8 @@ class Bazel(
 
             if (archiveOverride.urls.size > 1) {
                 logger.warn {
-                    "The module '${it.name}' has multiple URLs ${archiveOverride.urls} defined in the archive " +
-                        "override, but ORT only supports one URL for source artefacts. Only the first URL will be used."
+                    "The module '${it.name}' has multiple archive override URLs defined. Only the first URL of the " +
+                        "following URLs will be used: ${archiveOverride.urls.joinToString()}"
```

Comment:
While I agree with this change, I feel that the patches should be logged as it has an impact on compliance, even if ORT does not support them.
Could we create an issue with the patch urls here? https://github.com/oss-review-toolkit/ort/blob/1ec14b5256d38c849a07ad02dd7ab070b8635db5/plugins/package-managers/bazel/src/main/kotlin/Bazel.kt#L224

Reply:
Creating an issue would certainly be better / more visible than logging. Compliance people don't look at ORT logs in my experience, and even if they did, showing the patch's file names (or is it even the diff) IMO does not bring much value.

What's more important is that there *are* patches, i.e. the source code is modified compared to upstream, and that's exactly what 51c01ddbf013d6044ffd4774e2ca9c48e9fb4746 is about. IMO that's the proper way to signal to compliance people that they have to "watch out".

Regarding ORT not supporting patches, that's only true in the scanner context: I.e. ORT does not apply any patches before scanning, and will always scan the upstream source code without the patches. But that's only relevant if something that a scanner would detect gets modified by the patch. But mostly, such patches are rather small, and apply bug fixes or similar.

Reply:
> What's more important is that there are patches, i.e. the source code is modified compared to upstream, and that's exactly what https://github.com/oss-review-toolkit/ort/commit/51c01ddbf013d6044ffd4774e2ca9c48e9fb4746 is about. IMO that's the proper way to signal to compliance people that they have to "watch out".

Is `is_modified`  reflected in the WebApp report ?
Cf my question just underneath (I didn't know this property)

Reply:
> Is `is_modified` reflected in the WebApp report ?

I believe the Web App report does not show this (yet), but e.g. the `CycloneDxReporter` and `CtrlXAutomationReporter` do.

Reply:
So, are you fine with only making use of `isModified` and not logging any patches?

Reply:
I am fine with it if I can see the patch URLs in the ORT issues then. Deal ?

You don't have to implement it, a TODO is fine.

Reply:
I'm still not convinced that using issues is the right "communication channel" here. 

Instead, besides making `isModified=true` packages more visible in reports (for any package manager), we could think about further extending the data model to also track the actual patches (or better URLs to patches). That would be a more generic variant of https://github.com/oss-review-toolkit/ort/issues/8452 and also allow the scanner to apply the patches before scanning.